packages/mux-player/src/styles.css

```diff
@@ -25,6 +25,12 @@ media-control-bar [role="switch"] {
   height: 44px;
 }
 
+.size-extra-small media-control-bar [role="button"],
```

Comment:
interesting solution to "categorize" buttons. Might break down over time, but so would any `:is()` list or similar. 👌